### ISSUE TITLE
dbt_utils macros still using `dbt_utils.escape_single_quotes` 

### ISSUE THREAD

**Issue by zachoj10** [dbt-labs/dbt-utils#689]
### Describe the bug
I'm getting a warning when running dbt that the `escape_single_quotes` macro is now provided in dbt Core. From my digging around, it seems that there are some `dbt_utils` macros such as `pivot()` that are still relying on the version that's in dbt_utils instead of dbt Core. 

### Steps to reproduce

Running a dbt model that utilizes the pivot macro

### Expected results

No warnings from using a dbt_utils macro

### Actual results

```bash
Warning: the `escape_single_quotes` macro is now provided in dbt Core. It is no longer available in dbt_utils and backwards compatibility will be removed in a future version of the package. Use `escape_single_quotes` (no prefix) instead. The *** model triggered this warning.
```

### Screenshots and log output

![image](https://user-images.githubusercontent.com/1419736/192574922-78cd40b6-6e08-4515-b8e3-2cb2416dad77.png)

### System information
**The contents of your `packages.yml` file:**

```
packages:
  - package: dbt-labs/dbt_utils
    version: 0.9.1
```

**Which database are you using dbt with?**
- [ ] postgres
- [ ] redshift
- [ ] bigquery
- [x] snowflake
- [ ] other (specify: ____________)

**The output of `dbt --version`:**
```
Core:
  - installed: 1.2.1
  - latest:    1.2.1 - Up to date!

Plugins:
  - snowflake: 1.2.0 - Up to date!```


### Additional context

From a quick search, it seems that the only spot that `dbt_utils.escape_single_quotes` is referenced is in `pivot.sql` 

### Are you interested in contributing the fix?

Would definitely be interested in helping, I think I can put in a PR myself

**Comment by joellabes** [comment]
@zachoj10 this is fixed on the utils-v1 branch, but we might as well resolve it on the 0.9.x version as well. I'd welcome a PR!

**Comment by jsnb-devoted** [comment]
Could we cut a `0.9.3` branch and release a patch version with this in it? 
I made a branch of the `0.9.2` tag and add the commit here: https://github.com/jsnb-devoted/dbt-utils/tree/dev/0.9.3
Would love to get this in before the 1.0 release @joellabes

**Comment by bmoore813** [comment]
@joellabes poke ☝🏻

**Comment by jdub** [comment]
@joellabes Current version is 0.9.2 without this fix.

**Comment by joellabes** [comment]
Just shipped 0.9.5, but this missed the boat 😬 will handle in #728
